package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "app",
+  "private": "true",
+  "dependencies": {
+    "@babel/core": "^7.22.9",
+    "@babel/preset-env": "^7.22.9",
+    "@hotwired/stimulus": "^3.2.1",
+    "@hotwired/turbo-rails": "^7.3.0",
+    "@reduxjs/toolkit": "^1.9.5",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-redux": "^8.1.1",
+    "react-router-dom": "^6.14.2",
+    "redux": "^4.2.1",
+    "webpack": "^5.88.1",
+    "webpack-cli": "^5.1.4"
```

Comment:
- [ ] [OPTIONAL]
Axios package was used which was outside of the  _package.json_ Kindly include the axios dependency in the _package.json_ file to resolve the error below:

**Depency error:**
![image](https://github.com/OsvaldoBC/hello-rails-react/assets/12745474/76649397-2d95-45fd-9426-0c56ff03a339)